CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
-# Changelog
+-e # Changelog
```

Comment:
`-e`?

Reply:
might be an issue with the script generator - i''ll review this

Reply:
@NarmeenNaveedAhmed  would be removing this manually

Reply:
that was weird! fixed! :)